recipes/mlflow/meta.yaml

```diff
@@ -0,0 +1,81 @@
+{% set name = "mlflow" %}
+{% set version = "0.9.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c2c76d072f4735a229333095506acb3d6ec0305656ee7fc468e9b60a0e8d7b8a
+
+build:
+  number: 0
+  entry_points:
+    - mlflow=mlflow.cli:cli
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  noarch: python
+
+requirements:
+  host:
+    - python
+    - setuptools
+    - pip
+  run:
+    - click >=7.0
+    - cloudpickle >=0.6.1
+    - databricks-cli >=0.8.0
+    - docker-py >=3.6.0
+    - entrypoints
+    - flask
+    - gitpython >=2.1.0
+    - gunicorn
+    - mleap >=0.8.1
+    - numpy
+    - pandas
+    - pip
+    - protobuf >=3.6.0
+    - python
+    - python-dateutil
+    - pyyaml
+    - querystring_parser
+    - requests >=2.17.3
+    - scikit-learn
+    - scipy
+    - simplejson
+    - six >=1.10.0
+    - sqlparse
+    - boto3 >=1.7.12
+
+test:
+  imports:
+    - mlflow
+    - mlflow.azureml
+    - mlflow.entities
+    - mlflow.models
+    - mlflow.projects
+    - mlflow.protos
+    - mlflow.pyfunc
+    - mlflow.pytorch
+    - mlflow.rfunc
+    - mlflow.sagemaker
+    - mlflow.server
+    - mlflow.store
+    - mlflow.tracking
+    - mlflow.utils
+  commands:
+    - mlflow --help
+
+about:
+  home: https://mlflow.org/
+  license: Apache 2.0
+  license_family: APACHE
+  license_file: 
```

Comment:
either the pypi tarball contains a LICENSE file that can be referenced here or the LICENSE can be packaged as it was done in e.g. the databricks-cli package.